GRAPHICS/DisableOcclusionThisFrame.md

```diff
@@ -7,6 +7,17 @@ aliases: ["0x3669F1B198DCAA4F"]
 ```c
 // 0x3669F1B198DCAA4F 0x0DCC0B8B
 void DISABLE_OCCLUSION_THIS_FRAME();
-```
 
+This disables all occluders that cause the 'flickering' rendering under the map and inside buildings with no interiors, note that this may impact performance on lower end systems.
+```
```

Comment:
````suggestion
```

This disables all occluders that cause the 'flickering' rendering under the map and inside buildings with no interiors, note that this may impact performance on lower end systems.
````